amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -78,6 +69,12 @@ export class OwnerEditor extends React.Component<OwnerEditorProps, OwnerEditorSt
     this.inputRef = React.createRef();
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.itemProps !== this.props.itemProps) {
```

Comment:
nit: Given that `itemProps` is an object, does this check actually do anything or does it just evaluate to `true` all the time?

Reply:
I'm not really sure, I just ported over the behavior from getDerivedStateFromProps but I can check.